syncer/online-ddl-tools/ghost.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package syncer
+package onlineddl
```

Comment:
Suggest changing this directory's name to `onlineddl`

Reply:
there're other folders using `-` like `err-operator`, will rename later 😂